sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCSVScan.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2019-2024, NVIDIA CORPORATION.
+ * Copyright (c) 2019-2023, NVIDIA CORPORATION.
```

Comment:
I am working on updating the copyright years

Reply:
I have now reverted changes to copyright years for files that have currently been updated in 2024 other than from this PR